### PR TITLE
Create 2022-03-15-work-is-starting-SDK-SPID-CIE.md

### PR DESCRIPTION
**Pull request by ruphy**
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Create 2022-03-15-work-is-starting-SDK-SPID-CIE.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9e6bb9b6c0e73e41932381a002fdec4d3a05fe3d
</p>

File: _posts/en/2022-03-15-work-is-starting-SDK-SPID-CIE.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Work begins on unified SPID and CIE OpenID Connect SDKs."
+subtitle: "An important technological advancement that will make life easier for developers"
+date: 2022-03-15 10:00:00 +0200
+tags: developers sdk spid cie
+categories: news
+author: Giuseppe de Marco
+image: /assets/icons/logo-it.png
```
Comment (bfabio) [inline _posts/en/2022-03-15-work-is-starting-SDK-SPID-CIE.md:9]:
```suggestion
image: /assets/icons/logo-it.png
ref:
  it: /it/news/2022/03/15/iniziano-lavori-SDK-SPID-CIE
```

File: _posts/en/2022-03-15-work-is-starting-SDK-SPID-CIE.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- a clear documentation, in English language, also in the form of a README, on how to use the application, run the test units and contribute to the project;
+- an example project for demonstration purposes;
+- a docker image using the example project;
+- an automatic Continuous Integration (CI) pipeline with the test units;
+- an automatic CI pipeline with the official Quality Assessment tool used in the SPID and CIE id testing procedures;
+- a code test coverage of no less than 90%.
+
+We expect the SDKs to be finalised by mid-April.
+
+Would you like to actively contribute to the work in progress and help us test it? Would you like to propose a new SDK for your favourite programming language? Join us on the #spid-openid channel of [Slack Developers](https://slack.developers.italia.it/) and open issues and pull requests at:
```
Comment (bfabio) [inline _posts/en/2022-03-15-work-is-starting-SDK-SPID-CIE.md:43]:
```suggestion
Would you like to actively contribute to the work in progress and help us test it? Would you like to propose a new SDK for your favourite programming language? Join us on the [#spid-openid channel](https://slack.developers.italia.it/) and open issues and pull requests at:
```

**Review by bfabio — CHANGES_REQUESTED** [review]

**Comment by ruphy** [comment]
@bfabio :D

**Review by bfabio — APPROVED** [review]

**Pull request merged**